src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/custom.py

```diff
@@ -2,23 +2,99 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
+from __future__ import print_function
+import datetime
+
+
+# 1 hour in milliseconds
+DEFAULT_QUERY_TIME_RANGE = 3600000
+
+# ISO format with explicit indication of timezone
+DATE_TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
 
 def list_metric_definitions(client, resource_uri, metric_names=None):
     '''Commands to manage metric definitions.
     :param str resource_uri: The identifier of the resource
     :param str metric_names: The list of metric names
     '''
-    odata_filter = _list_metric_definitions_filter_builder(metric_names)
+    odata_filter = _metric_names_filter_builder(metric_names)
     metric_definitions = client.list(resource_uri, filter=odata_filter)
     return list(metric_definitions)
 
 
-def _list_metric_definitions_filter_builder(metric_names=None):
+def _metric_names_filter_builder(metric_names=None):
     '''Build up OData filter string from metric_names
     '''
     filters = []
     if metric_names:
         for metric_name in metric_names:
             filters.append("name.value eq '{}'".format(metric_name))
     return ' or '.join(filters)
+
+
+def list_metrics(client, resource_uri, time_grain,
+                 start_time=None, end_time=None, metric_names=None):
+    '''Lists the metric values for a resource.
+    :param str resource_uri: The identifier of the resource
+    :param str time_grain: The time grain. Granularity of the metric data returned in ISO 8601
+                           duration format, eg "duration'PT1M'"
+    :param str start_time: The start time of the query. In ISO format with explicit indication of
+                           timezone: 1970-01-01T00:00:00Z, 1970-01-01T00:00:00-0500
+    :param str end_time: The end time of the query. In ISO format with explicit indication of
+                         timezone: 1970-01-01T00:00:00Z, 1970-01-01T00:00:00-0500
+    :param str metric_names: The list of metric names
```

Comment:
Should clarify "The space separated list of metric names." Otherwise you will get people using comma separated values.